main.go

```diff
@@ -6,21 +6,25 @@ import (
 	"bufio"
 	"flag"
 	"fmt"
+	"io/ioutil"
 	"log"
 	"os"
 	"path/filepath"
 	"runtime"
+	"github.com/ghodss/yaml"
```

Comment:
Can you move this to the other 3rd party deps

Reply:
done.